clients/fides-js/src/components/ConsentModal.tsx

```diff
@@ -49,19 +51,8 @@ const ConsentModal = ({
             description={experience.description}
           />
         </p>
-        <GpcInfo />
+        {modalType !== "TCF" && <GpcInfo />}
         {children}
-        {experience.privacy_policy_link_label &&
```

Comment:
Moved this to its own component since we have more things to consider when rendering the link.